google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineCloudConfigurable.java

```diff
@@ -85,36 +81,25 @@ public JComponent createComponent() {
 
   @Override
   public boolean isModified() {
-    boolean isSdkValid = true;
-    try {
-      new CloudSdk.Builder()
-          .sdkPath(Paths.get(cloudSdkPanel.getCloudSdkDirectory()))
-          .build()
-          .validateCloudSdk();
-    } catch (AppEngineException aee) {
-      isSdkValid = false;
-    }
-
     // Forces a modify check so the user is unable to save an invalid Cloud SDK configuration from
     // Other Settings, on the Clouds menu.
-    return cloudSdkPanel.isModified() || !isSdkValid;
+    return cloudSdkPanel.isModified()
+        || !CloudSdkService.getInstance().isValidCloudSdk(cloudSdkPanel.getCloudSdkDirectory());
```

Comment:
doesn't the apply validation code protect against saving invalid cloud sdk path?